engine-nextgen/src/main/kotlin/graphql/nadel/enginekt/transform/NadelServiceTypeFilterTransform.kt

```diff
@@ -193,30 +194,30 @@ class NadelServiceTypeFilterTransform : NadelTransform<State> {
      *
      * Only use this to create subsets of OTHER lists. Never use this list directly.
      */
-    private fun getTypeNamesServiceOwns(
-        executionBlueprint: NadelOverallExecutionBlueprint,
-        service: Service,
-    ): Set<String> {
-        return service.underlyingSchema
-            .typeMap
-            .values
-            .asSequence()
-            .map {
-                executionBlueprint.getOverallTypeName(service, it.name)
-            }
-            .toSet()
-    }
-
-    private fun getUnderlyingTypeNamesServiceOwns(
-        executionBlueprint: NadelOverallExecutionBlueprint,
-        service: Service,
-        overallTypeNames: Set<String>
-    ): Set<String> {
-        return overallTypeNames
-            .asSequence()
-            .map {
-                executionBlueprint.getUnderlyingTypeName(service, it)
-            }
-            .toSet()
-    }
+    // private fun getTypeNamesServiceOwns(
+    //     executionBlueprint: NadelOverallExecutionBlueprint,
+    //     service: Service,
+    // ): Set<String> {
+    //     return service.underlyingSchema
+    //         .typeMap
+    //         .values
+    //         .asSequence()
+    //         .map {
+    //             executionBlueprint.getOverallTypeName(service, it.name)
+    //         }
+    //         .toSet()
+    // }
+    //
+    // private fun getUnderlyingTypeNamesServiceOwns(
+    //     executionBlueprint: NadelOverallExecutionBlueprint,
+    //     service: Service,
+    //     overallTypeNames: Set<String>
+    // ): Set<String> {
+    //     return overallTypeNames
+    //         .asSequence()
+    //         .map {
+    //             executionBlueprint.getUnderlyingTypeName(service, it)
+    //         }
+    //         .toSet()
+    // }
```

Comment:
I will clean up but I wanted to show the old code in PR terms when discussing the way is this different problem